### PR TITLE
feat(admin-client): list proposals

### LINKED ISSUE #126: Implement ProposalListFunction in defender-admin-client and expose `isActive`
There is no way to list proposals in `defender-admin-client` as of now. Looking at the response, it is also not easy as of now to identify active proposals, which would be a key user need.

The scope of this Issue is to:

1. Expose an endpoint for `ProposalListFunction` (as named in `template.yaml`) in `defender-admin-client`. 
2. Expose an `isActive` boolean on the response to this endpoint

### PR DESCRIPTION
**Pull request by pasevin**
This resolves #126 by introducing the `listProposals()` method allowing to retrieve a list of all (not archived) proposals.

### CODE REVIEW

File: packages/admin/README.md [hunk 1]
```diff
@@ -21,7 +20,7 @@ Start by creating a new _Team API Key_ in Defender, and granting it the capabili
 
 ```js
 const { AdminClient } = require('defender-admin-client');
-const client = new AdminClient({apiKey: API_KEY, apiSecret: API_SECRET});
+const client = new AdminClient({ apiKey: API_KEY, apiSecret: API_SECRET });
```
Comment (mok0230) [inline packages/admin/README.md:23]: Attention to detail in the docs! Love it

**Review by mok0230 — APPROVED** [review]
Looks great! Approved but of course has a dependency on https://github.com/OpenZeppelin/defender/pull/2755 so maybe wait to merge (could add `dont-merge` tag)

**Comment by zeljkoX** [comment]
@pasevin Is this one now ready to be merged?

**Comment by pasevin** [comment]
Yep as soon as the new Defender release comes out!

**Pull request merged**